infrastructure/main.tf

```diff
@@ -1,9 +1,22 @@
+provider "vault" {
+  address = "https://vault.reform.hmcts.net:6200"
+}
+
 locals {
   db_connection_options = "?ssl=true"
   ase_name        = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
   s2s_url         = "http://rpe-service-auth-provider-${var.env}.service.${local.ase_name}.internal"
 }
 
+// TODO: use the draft store test microservice once https://github.com/hmcts/service-auth-provider-app/pull/27 is merged
+data "vault_generic_secret" "tests_s2s_secret" {
```

Comment:
Would be better if we can move `tests_s2s_secret` and `test_idam_client_secret` to test config file

Reply:
You're probably right. I put it here as this bit retrieves data and test_config only outputs it. This split is unnecessary, though. Will change.